configs/j721s2_evm_gp_all_defconfig

```diff
@@ -10,7 +10,7 @@ K3IMGGEN_SOC=$(SOC_NAME)
 
 TFA_BOARD=generic
 
-OPTEE_PLATFORM=k3-j721s2
+OPTEE_PLATFORM=k3-j784s4
```

Comment:
do add a comment here specifying about this so that no one else thinks that this is a mistake